bycycle/utils/dataframes.py

```diff
@@ -55,10 +55,10 @@ def limit_df(df, fs, start=None, stop=None):
 
     start = 0 if start is None else start
 
-    df = df[df['sample_next_' + side_e].values >= start*fs]
+    df = df[df['sample_last_' + side_e].values >= start*fs]
```

Comment:
I'm guessing these changes should be removed because they're in #88 ?

Reply:
Good catch! The first two commits slipped in here by accident. I'll drop them.